packages/utils/src/TestingUtils/JestUtils/TestWrapper.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
+import { IntlProvider } from 'react-intl';
+
+const TestWrapper = ({ renderOptions = {}, children }) => {
+  const mockStore = configureStore();
+
+  return (
+    <IntlProvider locale="en">
+      <Provider store={renderOptions?.store || mockStore()}>
+        <MemoryRouter initialEntries={renderOptions?.initialEntries || ['/']}>
```

Comment:
MemoryRouter already has `initialEntries` defaulting to []: https://reactrouter.com/en/main/router-components/memory-router.

Let's also extract props for MemoryRouter and do simple spread like this:
```suggestion
        <MemoryRouter {...routerProps}>
```